coderd/database/migrations/000214_org_custom_role_array.up.sql

```diff
@@ -0,0 +1,4 @@
+-- Previous custom roles are now invalid, as the json changed. Since this is an
+-- experimental feature, there is no point in trying to save the perms.
+-- This does not elevate any permissions, so it is not a security issue.
```

Comment:
Might be no harm to call this out in case adventurous folks are already playing around with this.

Reply:
I added it in the PR description and the commit.